### PR TITLE
chore(plugins): adding plugins dir and application name

### PR DESCRIPTION
**Pull request by link108**
This creates the default plugins directory so plugins can be downloaded and placed locally. I needed to add the spring.application.name in gradle.properties as well, otherwise plugins would fail to start when running in a container.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -4,3 +4,4 @@ korkVersion=7.16.1
 spinnakerGradleVersion=7.0.1
 enablePublishing=false
 org.gradle.parallel=true
+spring.application.name=orca
```
Comment (robzienert) [inline gradle.properties]: Hmm, why is this necessary? [It should already set by the application itself](https://github.com/spinnaker/orca/blob/master/orca-web/src/main/groovy/com/netflix/spinnaker/orca/Main.groovy#L97).

Comment (link108) [reply gradle.properties]: I need to do some digging on this one, not sure why its needed, especially since its in the code already. When running in a container, plugins would fail to be found unless I added this.

Comment (marchello2000) [reply gradle.properties]: i also added this to tests here: https://github.com/spinnaker/orca/pull/3405

Comment (link108) [reply gradle.properties]: not needed, thank you both!
